cpp/src/io/utilities/column_buffer.cpp

```diff
@@ -51,19 +51,21 @@ std::unique_ptr<column> gather_column_buffer::make_string_column_impl(rmm::cuda_
   return make_strings_column(*_strings, stream, _mr);
 }
 
-void inline_column_buffer::allocate_strings_data(rmm::cuda_stream_view stream)
+void cudf::io::detail::inline_column_buffer::allocate_strings_data(rmm::cuda_stream_view stream)
```

Comment:
not sure why this was added
```suggestion
void inline_column_buffer::allocate_strings_data(rmm::cuda_stream_view stream)
```

Reply:
Yeah that was me being a little quick with the copy/paste.